### PR TITLE
package.json: remove peerDependencies

### PR DESCRIPTION
**Pull request by Mottie**
Fixes gh-548

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,9 +72,7 @@
     "url": "http://github.com/jquery/globalize/issues"
   },
   "dependencies": {
-    "cldrjs": "0.4.3"
-  },
-  "peerDependencies": {
+    "cldrjs": "0.4.3",
```
Comment (jzaefferer) [inline package.json]: This seems like the wrong fix. We want to keep the peerDependency, to allow users to install the version of their choice. We also want to install it when testing this project, so we should keep it in `peerDependencies`, while also adding it to `devDependencies`.

Comment (Mottie) [inline package.json]: After moving `"cldr-data": ">=25"` to `devDependencies` and adding a copy to `peerDependencies`, the warning message no longer appears. Do you want me to update the PR?

Comment (jzaefferer) [inline package.json]: Yes please

**Comment by rxaviers** [comment]
Hi @Mottie, a good news is that we removed cldr-data as a peer dependency (details in #638). Probably, longer than you expected though. Thanks for bringing it to us.

**Pull request closed without merging**